compiler/packages/babel-plugin-react-compiler/src/Optimization/OutlineJsx.ts

```diff
@@ -216,13 +216,22 @@ type OutlinedJsxAttribute = {
   place: Place;
 };
 
+let id = 1;
+function generateName(seen: Set<string>, oldName: string): string {
```

Comment:
this will cause non-deterministic output, it has to be scoped to the outer transform